Tests/Common/Securities/SubscriptionDataConfigTests.cs

```diff
@@ -40,5 +41,19 @@ public void UsedAsDictionaryKey()
             var config2 = new SubscriptionDataConfig(config1);
             Assert.IsFalse(set.Add(config2));
         }
+
+        [Test]
+        public void CanRemoveConsolidatorWhileEnumeratingList()
+        {
+            var config = new SubscriptionDataConfig(typeof(TradeBar), Symbols.SPY, Resolution.Minute, TimeZones.NewYork, TimeZones.NewYork, false, false, false, false, TickType.Trade, false);
+            var consolidator1 = new TradeBarConsolidator(1);
+            var consolidator2 = new TradeBarConsolidator(2);
+            config.Consolidators.Add(consolidator1);
+            config.Consolidators.Add(consolidator2);
+            foreach (var consolidator in config.Consolidators)
+            {
+                Assert.DoesNotThrow(() => config.Consolidators.Remove(consolidator));
```

Comment:
I think the `throw` would happen in the `foreach` loop 🤔

Reply:
Correct -- before the change this test fails immediately when it executes line 55. Following this change no exception is thrown. Maybe I'm misunderstanding your comment? :)

Reply:
My point was that I think it throws on line 53, not 55? don't think that `config.Consolidators.Remove(consolidator)` will throw - nit type of comment 😄

Reply:
ohh, it throws on ln55 w/out this fix due to modifying the collection via `Consolidators.Remove(consolidator)`